docs/firestore/android.md

```diff
@@ -1,6 +1,63 @@
 ---
-title: Android Setup
-description: Manually integrate Cloud Firestore into your Android application.
+title: Firestore Android Initialization
+description: Firestore initialization with Android.
 ---
 
 # Android Manual Linking
+
+> The following steps are only required if your environment does not have access to React Native
```

Comment:
Can we change this to:

```
The following steps are only required if you are using React Native <= 0.59 or need to manually integrate the library.
```

Reply:
Sure! Updated on android.md and ios.md 😄